app/actions/definitions/teams.tsx

```diff
@@ -1,40 +1,62 @@
+import { PlusIcon } from "outline-icons";
 import * as React from "react";
 import styled from "styled-components";
+import TeamNew from "~/scenes/TeamNew";
 import { createAction } from "~/actions";
 import { loadSessionsFromCookie } from "~/hooks/useSessions";
 
 export const changeTeam = createAction({
-  name: ({ t }) => t("Switch team"),
+  name: ({ t, isContextMenu }) =>
+    isContextMenu ? t("Teams") : t("Switch team"),
   placeholder: ({ t }) => t("Select a team"),
   keywords: "change workspace organization",
-  section: "Account",
-  visible: ({ currentTeamId }) => {
-    const sessions = loadSessionsFromCookie();
-    const otherSessions = sessions.filter(
-      (session) => session.teamId !== currentTeamId
-    );
-    return otherSessions.length > 0;
+  section: "Team",
+  visible: ({ currentTeamId, isContextMenu }) => {
+    return isContextMenu || getOtherSessions(currentTeamId).length > 0;
   },
-  children: ({ currentTeamId }) => {
-    const sessions = loadSessionsFromCookie();
-    const otherSessions = sessions.filter(
-      (session) => session.teamId !== currentTeamId
-    );
+  children: ({ currentTeamId, isContextMenu }) => {
+    return [
+      ...getOtherSessions(currentTeamId).map((session) => ({
+        id: session.url,
+        name: session.name,
+        section: "Team",
+        icon: <Logo alt={session.name} src={session.logoUrl} />,
+        perform: () => (window.location.href = session.url),
+      })),
+      ...(isContextMenu ? [createTeam] : []),
+    ];
+  },
+});
 
-    return otherSessions.map((session) => ({
-      id: session.url,
-      name: session.name,
-      section: "Account",
-      icon: <Logo alt={session.name} src={session.logoUrl} />,
-      perform: () => (window.location.href = session.url),
-    }));
+const createTeam = createAction({
+  name: ({ t }) => t("New team..."),
+  keywords: "change switch create workspace organization",
+  section: "Team",
+  icon: <PlusIcon />,
+  perform: ({ t, event, stores }) => {
+    event?.preventDefault();
+    event?.stopPropagation();
+    const { user } = stores.auth;
+    user &&
+      stores.dialogs.openModal({
+        title: t("Create a team"),
+        content: <TeamNew user={user} />,
+      });
   },
 });
 
+function getOtherSessions(currentTeamId = "") {
+  const sessions = loadSessionsFromCookie();
+  const otherSessions = sessions.filter(
+    (session) => session.teamId !== currentTeamId
+  );
+  return otherSessions;
+}
+
 const Logo = styled("img")`
   border-radius: 2px;
   width: 24px;
   height: 24px;
 `;
 
-export const rootTeamActions = [changeTeam];
+export const rootTeamActions = [changeTeam, createTeam];
```

Comment:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/427579/184395248-4af8a048-3b6f-4454-b24a-c5bbcd6ad0a6.png">
I think it's nicer to have the "new team" item be top level in the command menu, but the logic is slightly more annoying

Reply:
it's nested in the context menu 
<img width="382" alt="image" src="https://user-images.githubusercontent.com/427579/184395466-96b9c1e9-35ea-4b56-ac7b-79227ce1b808.png">